README.md

```diff
@@ -27,79 +27,56 @@ the specified user.
 
 ## Installation
 
+[Razee Deploy Delta](https://github.com/razee-io/razeedeploy-delta) is the
+recommended way to install this webhook.
+
 Communication between admission controllers and webhooks is via HTTPS:
 
 - Webhook servers must present a valid certificate. The certificate and key are
-  stored in a secret.
+  stored in `impersonation` secret.
 - Admission controllers will validate the presented certificate by using the
-  certificate authority set in `caBundle` field in `MutatingWebhookConfiguration`.
+  certificate authority set in `caBundle` field in `impersonation-webhook`
+  MutatingWebhookConfiguration.
 
-It's up to the user to create and manage the certificate, key, certificate
-authority, and rotation strategy.
+Note: [Razee Deploy Delta](https://github.com/razee-io/razeedeploy-delta) will
+deploy the webhook with a self-signed certificate by default. Custom
+certificate can be used by updating `impersonation` secret and
+`impersonation-webhook` MutatingWebhookConfiguration.
 
 For your reference, a self-signed certificate can be quickly created with the
 following command for demo purpose:
 
 ```bash
 openssl req -x509 -newkey rsa:4096 -sha256 -days 3650 -nodes -keyout server.key 
--out server.crt -subj "/CN=impersonation-webhook.default.svc" 
--addext "subjectAltName=DNS:impersonation-webhook.default.svc"
+-out server.crt -subj "/CN=impersonation-webhook.{NAMESPACE}.svc" 
+-addext "subjectAltName=DNS:impersonation-webhook.{NAMESPACE}.svc"
+```
+
+## Customize the webhook
+
+The optional `razeedeploy-config` ConfigMap can be used to customize the
+webhook.
+
+Because the ConfigMap is optional, if it is created the first time, you must
+restart webhook pods, so the deployment can mount the ConfigMap
+as a volume.
```

Comment:
```suggestion
Because the ConfigMap is optional, if it is created after the ImpersonationWebhook
deployment, you must restart ImpersonationWebhook deployment's
pods, so the deployment can mount the ConfigMap as a volume.
```